### ISSUE TITLE
No Dashboard or Report Graphics are Generated

### ISSUE THREAD

**Issue by Hyperspeed1313** [moneymanagerex/moneymanagerex#6258]
MMEX version:
 - [x] 1.6.4
 - [x] 1.7.0
 - [ ] 1.7.1
 - [ ] 1.7.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [ ] Windows
 - [ ] Mac OSX
 - [x] Linux (Fedora 38)

# Description of the bug

The Dashboard and all predefined reports in the sidebar are completely blank.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/2414807/b85b81c5-9ede-4f3f-b053-88c8c3e79ed0)

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:
Install MMEX from Flathub, import transactions from .qif file. Then go to the Dashboard or any of the predefined reports.
Also reproducible in latest beta build at time of writing (1.7.0-Beta.1 built Oct 22 2023 09:03:39, commit c5c75b37b)

Expected result:
Some kind chart or summary appears

Actual result:
Blank window

# Additional information
Console Output:
On startup
```
$ flatpak run org.moneymanagerex.MMEX 
Gtk-Message: 15:39:49.612: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:39:49.612: Failed to load module "pk-gtk-module"
Gtk-Message: 15:39:49.617: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:39:49.617: Failed to load module "pk-gtk-module"
```
On finshing setting up new database or loading an existing database (loads into dashboard):
```
Gtk-Message: 15:41:18.745: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:41:18.745: Failed to load module "pk-gtk-module"
Gtk-Message: 15:41:18.760: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:41:18.760: Failed to load module "pk-gtk-module"
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1350x1191: Permission denied
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1350x1191: Permission denied
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1350x1191: Permission denied
Failed to create EGL images for DMABufs with file descriptors -1, -1 and -1
```
On opening any of the predefined reports, or switching back to the dashboard
```
** (mmex:2): CRITICAL **: 15:43:34.779: Cannot register URI scheme memory more than once
Gtk-Message: 15:43:34.846: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:43:34.846: Failed to load module "pk-gtk-module"
Gtk-Message: 15:43:34.855: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:43:34.855: Failed to load module "pk-gtk-module"
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1645x912: Permission denied
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1645x912: Permission denied
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1645x912: Permission denied
Failed to create EGL images for DMABufs with file descriptors -1, -1 and -1
```
Specific to the 1.7.0 beta build, the following line also prints when switching to a predefined report or the dashboard, just after the critical print:
```
** (process:77775): WARNING **: 15:51:01.523: Error writing credentials to socket: Error sending message: Broken pipe
```

**Comment by Hyperspeed1313** [comment]
Closing as duplicate of #6214
